### PR TITLE
tests: Split up ArrayOOBGraphics

### PR DESCRIPTION
**Pull request by jeremyg-lunarg**
This test case is huge and hard to debug, so split it into a bunch of smaller tests.

### CODE REVIEW

File: tests/unit/gpu_av_descriptor_indexing.cpp [hunk 1]
```diff
@@ -15,6 +15,7 @@
 #include "../framework/pipeline_helper.h"
 #include "../framework/descriptor_helper.h"
 #include "../framework/gpu_av_helper.h"
+#include <iostream>
```
Comment (jeremyg-lunarg) [inline tests/unit/gpu_av_descriptor_indexing.cpp:18]: there's a bunch of debug prints in here for now. I'd like to convert them into some sort of gtest message at some point.

Comment (arno-lunarg) [reply tests/unit/gpu_av_descriptor_indexing.cpp:18]: are they gone now, or am I just not seeing them??

Comment (arno-lunarg) [reply tests/unit/gpu_av_descriptor_indexing.cpp:18]: Ah I guess that there are the `SCOPED_TRACE` calls?

**Review by arno-lunarg — APPROVED** [review]

File: tests/unit/gpu_av_descriptor_indexing.cpp (hunk 1 above)
Comment (jeremyg-lunarg) [reply tests/unit/gpu_av_descriptor_indexing.cpp:18]: yes, thats the best thing I could find in gtest. I still need to repush to remove this header.

**Review by spencer-lunarg — APPROVED** [review]

**Pull request merged**